projects/exui-common-lib/src/lib/components/write-address/write-address.component.html

```diff
@@ -3,16 +3,15 @@
     <span class="govuk-visually-hidden">Error:</span>{{optionErrorMessage}}
   </p>
   <div *ngIf="postcodeLookupVisible()">
-    <div class="govuk-body postcodeLookup" id="addressLookup">
+    <span class="govuk-label"><b>Provide address details</b></span>
```

Comment:
@connorpgpmcelroy - `<b>` tag is deprecated; use `<strong>` instead (see https://www.w3schools.com/tags/tag_b.asp)